3_visualize/src/plot_timeseries.R

```diff
@@ -1,8 +1,10 @@
 # This function creates a time series of the NWIS site data by station_name and during 
 # globally-defined start and end dates.
-plot_nwis_timeseries <- function(fileout, site_data_styled, width = 12, height = 7, units = 'in'){
+plot_nwis_timeseries <- function(fileout, in_file, width = 12, height = 7, units = 'in'){
   
-  ggplot(data = site_data_styled, aes(x = dateTime, y = water_temperature, color = station_name)) +
+  load(in_file)
```

Comment:
Following up from my comment above about RDS vs .Rdata - this would change to `site_data_processed <- readRDS(in_file)` (or you could leave the argument as `site_data_styled` as you had it before.